imod/tests/test_msw/test_copy_files.py

```diff
@@ -0,0 +1,69 @@
+from pytest_cases import parametrize_with_cases
+
+from imod.msw.copy_files import CopyFiles
+
+
+def setup_src_files(directory, filenames):
```

Comment:
You could call this create_test_files or write_test_files

Reply:
Good point, done!